bin/pyenv-installer

```diff
@@ -89,4 +89,11 @@ if ! command -v pyenv 1>/dev/null; then
       ;;
     esac
   } >&2
+elif [ -d "${PYENV_ROOT}" ]; then
```

Comment:
If I understand this correctly, the way this is implemented it will only show this error if pyenv is corrctly installed already (or at least the command being available). 

The error though is suppposed to happen, when you try to install pyenv and there is a `.pyenv` directory already.

So I think this check should actually be at the very beginning. If a .pyenv directory already exists, the installer should crash with a helpful error right away before doing anything.

Reply:
Noted, on it.

Reply:
@obestwalter done.